wandb/sdk/lib/wburls.py

```diff
@@ -0,0 +1,42 @@
+"""Container for urls used in the wandb package.
+
+Use this anytime a URL is displayed to the user.
+
+Usage:
+    ```python
+    from wandb.sdk.lib.wburls import wburls
+
+    print(f"This is a url {wburls.get('cli_launch')}")
+    ```
+"""
+
+from typing import Dict, Optional
+from typing import TYPE_CHECKING
+
+
+if TYPE_CHECKING:
+    from ._wburls_generated import URLS
+
+
+class WBUrls:
```

Comment:
nit: feel weird that the url is not all capitalized

Reply:
yeah,  but then it is all caps and flake and other tools complain that you shouldnt make your thing all caps.  the problem with using a class name with two initialisms :)

Reply:
what about `WBURLs`? no strong feelings... :)

Reply:
SURE!  done